simphony_mayavi/cuds/vtk_lattice.py

```diff
@@ -175,16 +261,18 @@ def from_lattice(cls, lattice):
             y, z, x = numpy.meshgrid(
                 range(size[1]), range(size[2]), range(size[0]))
             indices = izip(x.ravel(), y.ravel(), z.ravel())
-        elif lattice_type == 'Hexagonal':
-            x, y, z = numpy.meshgrid(
-                range(size[0]), range(size[1]), range(size[2]))
+        elif lattice_type in BravaisLattice:
```

Comment:
@dpinte In this block, it handles lattice types such as hexagonal, rhombohedral, body-centered cubic, monoclinic ... etc., basically any lattice type that is defined in the BravaisLattice class except for cubic/tetragonal/orthorhombic.  The cubic/tetragonal/orthorhombic types are isolated in the `if` block because they can be represented by tvtk.ImageData, which is more efficient than PolyData.  Would adding comments explaining the above be sufficient?